scripts/ci/lib.sh

```diff
@@ -1739,6 +1739,13 @@ _EO_SUITE_HEADER_
         local result="${lines[1]}"
         local details="${lines[2]}"
 
+        # XML escape description
+        description="${description//&/&amp;}"
+        description="${description//\"/&quot;}"
+        description="${description//\'/&#39;}"
+        description="${description//</&lt;}"
+        description="${description//>/&gt;}"
```

Comment:
I think we should just drop this chars

Reply:
I can think of some messages when dropping these chars would make things rather unclear...

Reply:
I'm leaning towards what @janisz says i.e. drop but I'll force a similar failure to the original issue and see how it looks in the JUNIT -> JIRA/Slack test pipeline.

Reply:
It looks good in [Junit](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/stackrox_stackrox/10263/pull-ci-stackrox-stackrox-master-gke-qa-e2e-tests/1765868946835116032) and [Slack](https://redhat-internal.slack.com/archives/C068WMY4SEQ/p1709857158076569). 
Not so good in [JIRA](https://issues.redhat.com/browse/RS-641) but we can work on that separately.